Leipzig hackathon: No cash at cafeteria (#1647)
nf-core/nf-co.re: markdown/events/2023/hackathon-march-2023/germany-mpi-eva.md

## Organisational Information

> MPI-EVA has an in-house cafeteria for lunch, and accepts both EC or credit card. Cash is **no**t possible!

Reviewer: ```suggestion
MPI-EVA has an in-house cafeteria for lunch, and accepts both EC or credit card. Cash is **not** possible!
```
